bims/static/js/views/detail_dashboard/site_detail.js

```diff
@@ -1170,18 +1184,14 @@ define([
                     datasets.push({
                         label: key_item,
                         data: graph_data,
-                        backgroundColor: [
-                            background_colour[idx]
-                        ],
-                        borderColor: [
-                            background_colour[idx]
-                        ],
+                        backgroundColor: 'lightgrey',
```

Comment:
please change the color to this
```suggestion
                        backgroundColor: '#cfdeea',
```